themes/default/content/resources/getting-started-with-infrastructure-as-code-on-digital-ocean/index.md

```diff
@@ -69,8 +69,9 @@ main:
     # case-sensitive
     tags:
         level: Beginner # Beginner, Intermediate, Advanced
-        topics: ["DigitalOcean"]
+        topics: []
         languages: []
+        clouds: ["DigitalOcean"]
```

Comment:
Why is it when I choose this, I get no results? 

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/39aeb925-7283-4723-92c7-507089e3bf09)

I get this with others as well, like CodeFresh in the Topics menu, Azure, etc. Is this because they're in the past, and that's preventing them from displaying?

Reply:
The filters are OR'd except for upcoming/on-demand (since the content lives under tabs) so the only DigitalOcean content shows up under the on-demand tab.

Reply:
Oh -- so the tabs behave like a filter also, then. That seems like a design bug -- makes it look like we don't have content we actually do. 😬

We don't need to block on it, but we should file a follow-up issue for it, as I'm sure someone else will bump into it and get the wrong impression. Unless you can think of an easy solution to include in this PR?

Reply:
I don't really have a solution in mind so maybe we need some UX support? I created a [follow-up issue](https://github.com/pulumi/pulumi-hugo/issues/4175) for this